submissions/ayodeleayobami/landingpage/style.css

```diff
@@ -0,0 +1,85 @@
+/*import fonts from google fonts*/
+@import url('https://fonts.googleapis.com/css?family=Anton|Gugi|Lato|Roboto|Sofia&display=swap');
+*{
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font-family: Lato;
+}
+.container{
```

Comment:
Please work on the following on your css files:
- Leave a whitespace between the selectors and the curl braces.
- Leave a space between css blocks
This will make your code easier to read and maintain.

Reply:
Done